client/state/sites/domains/selectors.js

```diff
@@ -65,12 +65,14 @@ export function getDecoratedSiteDomains( state, siteId ) {
 		return {
 			...domain,
 
-			// Add registration moment from registrationDate
```

Comment:
🙇 Thanks for the explanation! Missed the fact that those aren't used for the same data. :(